Makefile

```diff
@@ -14,6 +14,14 @@ SOURCEDIRS := ./cmd ./pkg
 PACKAGES   := $(shell go list $(addsuffix /...,$(SOURCEDIRS)))
 SOURCES    := $(shell find $(SOURCEDIRS) -name '*.go' -print)
 
+IMAGE_BUILD_CMD ?= podman build
```

Comment:
Can we use a similar approach like - https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/install/runtime-payload/Makefile#L4-L5

This will be uniform and help us create multi-arch images going forward, which will be useful